### PR TITLE
LazyView Docs

### PR DESCRIPTION
**Pull request by pictos**
## What changes to the docs does this PR provide?

This PR adds the docs for the LazyView Control.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] For new pages, used the [provided template](https://github.com/MicrosoftDocs/xamarin-communitytoolkit/blob/master/docs/.template.md).
- [x] For new features, added an entry in the [Table of Contents](https://github.com/MicrosoftDocs/xamarin-communitytoolkit/blob/master/docs/toc.yml).
- [x] Ran against a spell and grammar checker.
- [x] Contains **NO** breaking changes.

### CODE REVIEW

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[f3b33eb](https://github.com/pictos/xamarin-communitytoolkit/commits/f3b33ebb080fe9a5d68fcd3aafd9cc96af3d92ad)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/views/lazyview.md](https://github.com/pictos/xamarin-communitytoolkit/blob/lazyView/docs/views/lazyview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/views/lazyview?branch=pr-en-us-78) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2021%5C2%5C11%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202102112341048331-78%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=OMSlhELB5iaavbDg9sE3hdIWHmuzfshPsb877deIl08%3D&st=2021-02-11T23%3A36%3A40Z&se=2021-03-14T23%3A41%3A40Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by TheCodeTraveler** [comment]
Hey @pictos!

When you get a chance, please add `LazyView` to the Table of Contents, here:
https://github.com/MicrosoftDocs/xamarin-communitytoolkit/blob/0b62f7199b985f3a8dd8bdcefedd82c90eada473/docs/TOC.yml#L101-L124

**Comment by pictos** [comment]
@brminnick thanks for the tip (;

File: docs/TOC.yml [hunk 1]
```diff
@@ -122,5 +122,7 @@
       href: views/tabview.md
     - name: "UniformGrid"
       href: views/uniformgrid.md
+    - name: CharactersValidationBehavior
+      href: behaviors/charactersvalidationbehavior.md
```
Comment (Cfun1) [inline docs/TOC.yml]:
```suggestion
    - name: LazyView
      href: views/lazyview.md
```
Shouldn't be ?

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[4049fa2](https://github.com/pictos/xamarin-communitytoolkit/commits/4049fa2864bb0839e9c01167fc68015b66f4d200)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-1d53e204)
[]() | :x:Error | | [Details](#user-content-1d53e204)

<a id="1d53e204"></a>
###
 - **[Error-RuningBuildFailed]** `````Some unexpected errors happened when running build, please open a ticket in https://SiteHelp and include the error report for our team to troubleshoot`````
 - **Line 0, Column 0**: **[Warning-validation-incomplete]** `````Failed to get the validation ruleset and validation was not completed. This happens when there's an issue with the service and continuing to retry the call could cause build delays. You might have content issues that were not reported. To retry validation, close and re-open your PR, or rebuild your branch via Docs Portal (requires admin permissions). If you need admin help or if you continue to see this message, file an issue via https://SiteHelp.`````

<a id="1d53e204"></a>
###
 - **[Error-RuningBuildFailed]** `````Some unexpected errors happened when running build, please open a ticket in https://SiteHelp and include the error report for our team to troubleshoot`````
 - **Line 0, Column 0**: **[Warning-validation-incomplete]** `````Failed to get the validation ruleset and validation was not completed. This happens when there's an issue with the service and continuing to retry the call could cause build delays. You might have content issues that were not reported. To retry validation, close and re-open your PR, or rebuild your branch via Docs Portal (requires admin permissions). If you need admin help or if you continue to see this message, file an issue via https://SiteHelp.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2021%5C2%5C12%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202102120110118702-78%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=LNmKbuwcJ3V3zZdq3YsZuK%2FApr0nCc3rbaBW4Q%2FzOxs%3D&st=2021-02-12T01%3A13%3A59Z&se=2021-03-15T01%3A18%3A59Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://teams.microsoft.com/l/message/19:7ecffca1166a4a3986fed528cf0870ee@thread.skype/1590030602688?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&parentMessageId=1590030602688&teamName=Docs%20Support&channelName=General&createdTime=1590030602688) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[754607c](https://github.com/pictos/xamarin-communitytoolkit/commits/754607c1ff74781e74acec505886dcc6a05fe46d)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-445f22ab)
[]() | :x:Error | | [Details](#user-content-445f22ab)

<a id="445f22ab"></a>
###
 - **[Error-RuningBuildFailed]** `````Some unexpected errors happened when running build, please open a ticket in https://SiteHelp and include the error report for our team to troubleshoot`````
 - **Line 0, Column 0**: **[Warning-validation-incomplete]** `````Failed to get the validation ruleset and validation was not completed. This happens when there's an issue with the service and continuing to retry the call could cause build delays. You might have content issues that were not reported. To retry validation, close and re-open your PR, or rebuild your branch via Docs Portal (requires admin permissions). If you need admin help or if you continue to see this message, file an issue via https://SiteHelp.`````

<a id="445f22ab"></a>
###
 - **[Error-RuningBuildFailed]** `````Some unexpected errors happened when running build, please open a ticket in https://SiteHelp and include the error report for our team to troubleshoot`````
 - **Line 0, Column 0**: **[Warning-validation-incomplete]** `````Failed to get the validation ruleset and validation was not completed. This happens when there's an issue with the service and continuing to retry the call could cause build delays. You might have content issues that were not reported. To retry validation, close and re-open your PR, or rebuild your branch via Docs Portal (requires admin permissions). If you need admin help or if you continue to see this message, file an issue via https://SiteHelp.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2021%5C2%5C12%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202102120126000244-78%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=UulPgyA%2BdQtwE6GvrOyAZKOezLP44TBmSyUwVmB5buM%3D&st=2021-02-12T01%3A32%3A19Z&se=2021-03-15T01%3A37%3A19Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://teams.microsoft.com/l/message/19:7ecffca1166a4a3986fed528cf0870ee@thread.skype/1590030602688?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&parentMessageId=1590030602688&teamName=Docs%20Support&channelName=General&createdTime=1590030602688) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[6cc3626](https://github.com/pictos/xamarin-communitytoolkit/commits/6cc36265f0d545615fd502fb60d18f89129a2ad1)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-30e17ef8)
[]() | :x:Error | | [Details](#user-content-30e17ef8)

<a id="30e17ef8"></a>
###
 - **[Error-RuningBuildFailed]** `````Some unexpected errors happened when running build, please open a ticket in https://SiteHelp and include the error report for our team to troubleshoot`````
 - **Line 0, Column 0**: **[Warning-validation-incomplete]** `````Failed to get the validation ruleset and validation was not completed. This happens when there's an issue with the service and continuing to retry the call could cause build delays. You might have content issues that were not reported. To retry validation, close and re-open your PR, or rebuild your branch via Docs Portal (requires admin permissions). If you need admin help or if you continue to see this message, file an issue via https://SiteHelp.`````

<a id="30e17ef8"></a>
###
 - **[Error-RuningBuildFailed]** `````Some unexpected errors happened when running build, please open a ticket in https://SiteHelp and include the error report for our team to troubleshoot`````
 - **Line 0, Column 0**: **[Warning-validation-incomplete]** `````Failed to get the validation ruleset and validation was not completed. This happens when there's an issue with the service and continuing to retry the call could cause build delays. You might have content issues that were not reported. To retry validation, close and re-open your PR, or rebuild your branch via Docs Portal (requires admin permissions). If you need admin help or if you continue to see this message, file an issue via https://SiteHelp.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2021%5C2%5C12%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202102120139565429-78%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=vJJG0dl9gwlP%2BtXgpCGw5%2B0Q0ZeGArHxgRjo2Sycmu0%3D&st=2021-02-12T01%3A42%3A39Z&se=2021-03-15T01%3A47%3A39Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://teams.microsoft.com/l/message/19:7ecffca1166a4a3986fed528cf0870ee@thread.skype/1590030602688?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&parentMessageId=1590030602688&teamName=Docs%20Support&channelName=General&createdTime=1590030602688) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[5600662](https://github.com/pictos/xamarin-communitytoolkit/commits/560066230a4bde401011da3dc1d2d16dc364e5e8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.yml](https://github.com/pictos/xamarin-communitytoolkit/blob/lazyView/docs/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/views/lazyview?branch=pr-en-us-78) |
[docs/views/lazyview.md](https://github.com/pictos/xamarin-communitytoolkit/blob/lazyView/docs/views/lazyview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/views/lazyview?branch=pr-en-us-78) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2021%5C2%5C12%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202102121033282704-78%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ncau4Bwz2o2pLS%2Fr7lRi%2BIJF2XMq6o9NRe6YaGoi1RU%3D&st=2021-02-12T10%3A29%3A02Z&se=2021-03-15T10%3A34%3A02Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by davidbritch — APPROVED** [review]
Thanks @pictos

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[8ebc8e1](https://github.com/pictos/xamarin-communitytoolkit/commits/8ebc8e147efc6ecd2a21a47d78528ae5ef50a2ac)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.yml](https://github.com/pictos/xamarin-communitytoolkit/blob/lazyView/docs/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/views/lazyview?branch=pr-en-us-78) |
[docs/views/lazyview.md](https://github.com/pictos/xamarin-communitytoolkit/blob/lazyView/docs/views/lazyview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/views/lazyview?branch=pr-en-us-78) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2021%5C2%5C12%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202102121040575084-78%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=bMbY5WZkx2J4pmdz3EmBFjNkYPCCCVnUtPQABBRk24I%3D&st=2021-02-12T10%3A36%3A36Z&se=2021-03-15T10%3A41%3A36Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[a7fae96](https://github.com/pictos/xamarin-communitytoolkit/commits/a7fae96a63e5acf0df9929d2e074709dd04cd578)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.yml](https://github.com/pictos/xamarin-communitytoolkit/blob/lazyView/docs/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/views/lazyview?branch=pr-en-us-78) |
[docs/views/lazyview.md](https://github.com/pictos/xamarin-communitytoolkit/blob/lazyView/docs/views/lazyview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/views/lazyview?branch=pr-en-us-78) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2021%5C2%5C12%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202102121043076292-78%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=OBJFYPqOD2VEsV%2BKjl%2BvHBny2bHn7Ir7mozm20OMSQ0%3D&st=2021-02-12T10%3A38%3A39Z&se=2021-03-15T10%3A43%3A39Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**